third_party/rust/Cargo.lock

```diff
@@ -363,7 +363,7 @@ dependencies = [
  "unicode-linebreak",
  "url",
  "urlencoding",
- "uuid 0.8.2",
+ "uuid 1.3.0",
```

Comment:
this probably doesn't break anything, but cargo drops the version when there is only one so at the very least this will trigger a change the next time gnrt is run and might be confusing to whoever runs it

Reply:
Oh, nice. I'd never understood what made the explicit version numbers come and go.